src/Microsoft.TemplateEngine.Cli/TemplateResolution/TemplateGroupMatchInfo.cs

```diff
@@ -49,7 +49,7 @@ internal class TemplateGroupMatchInfo
         /// <summary>
         /// True when the group matches group filters.
         /// </summary>
-        internal bool IsGroupMatch => _groupDispositions.Any(x => x.Kind != MatchKind.Mismatch);
+        internal bool IsGroupMatch => _groupDispositions.All(x => x.Kind != MatchKind.Mismatch);
```

Comment:
Can you explain what is happening here?

Reply:
will try

`IsGroupMatch` defines match condition on group filters. For most of cases there is only one filter - on short name, so Any/All doesn't matter. However for help 2 (language and short name) is defined, therefore for no match/ambiguous choice where we listed matching we displayed too many groups - match on language was enough. Interesting enough instantiate was not really affected by this bug, likely we never use affected properties for this case.